docs/custom_promise_types/cfengine.py

```diff
@@ -141,14 +141,32 @@ def _log(self, level, message):
         self._out.write(f"log_{level}={message}\n")
         self._out.flush()
 
+    def log_critical(self, message):
```

Comment:
What are warning and critical for? Unused right?

Reply:
https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#log-levels